src/store/ethereum/index.ts

```diff
@@ -278,12 +278,24 @@ export async function approve(context: ActionContext, payload: Transfer) {
     erc20Contracts.get(symbol),
     "Contract not initialized",
   )
-  await contract.methods
-    .approve(to, weiAmount.toString())
-    // @ts-ignore
-    .send({
-      from: context.state.address,
-    })
+  try {
+    const result = await contract.methods
+      .approve(to, weiAmount.toString())
+      // @ts-ignore
+      .send({
+        from: context.state.address,
+      })
+    return result
```

Comment:
Why is this function returning something now when it didn't return anything before (other than `Promise<void>`)?

Reply:
send returns Promise<hash> no?

Reply:
No idea, I just see it was returning `Promise<void>` before, and now it returns whatever `.send` does OR `""`... which is odd. Also what's with the `@ts-ignore`?